apps/src/lib/levelbuilder/lesson-editor/ActivitiesEditor.jsx

```diff
@@ -145,17 +145,28 @@ class ActivitiesEditor extends Component {
             allowMajorCurriculumChanges={allowMajorCurriculumChanges}
           />
         ))}
-        {this.props.hasLessonPlan && (
+        <div>
+          {this.props.hasLessonPlan && (
+            <button
+              onMouseDown={this.handleAddActivity}
+              className="btn add-activity"
+              style={styles.addActivity}
+              type="button"
+            >
+              <i style={{marginRight: 7}} className="fa fa-plus-circle" />
+              Activity
+            </button>
+          )}
           <button
```

Comment:
This should be a link, possibly styled as a button, as it'll take the user to a new page (likely in a new tab). 

Also curious on your thoughts on this button/link linking to the new rubric page immediately? I agree that dynamically linking to new or edit is out of scope though

Reply:
Thanks - I was able to read up on the button/Button/a-link discussion on the accessibility channel and made this change.

I did make it link to the new rubric page immediately here, too.  E